woocommerce/woocommerce-gutenberg-products-block
# Convert Product SKU block to TypeScript

#### Description

Converting the Product SKU component from JS to TypeScript.

**Note:** this PR builds on the changes to the shared config that are being introduced in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4891. Once said PR is merged we'll be able to set the base for this PR to `trunk`.

### Testing

#### Automated Tests

### Manual Testing

Please regression test this component to ensure it is still working as expected. You can also do this by comparing the behavior to that on the `trunk` branch.

1. Check out the branch and check that the compiler reports no TS errors associated with the converted files.
2. Add block(s) to your page which can use the component. Such as the `Single Product` or `All Products` block for example (the more variety the better).
2. Test no regressions with the Product SKU component's behavior have been introduced.

### Changelog

> Product SKU: Convert block to TypeScript
